### PR TITLE
Loan transactions

### PR DESCRIPTION
**Pull request by s-ahmed-codat**
# Description

Add loan transactions endpoint documentation

## Type of change

Please delete options that are not relevant.

- [ ] New document(s)/updating existing

### CODE REVIEW

File: docs/assess/loans/loan-transactions [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+createdAt: "2023-06-15T10:32:29.917Z"
+updatedAt: "2023-06-15T16:35:29.376Z"
+---
+
+## Introduction to Loan Transactions endpoint
+Loan data is often not uniform, spread across a variety of integrations and of different types.
+These loans are often hard to identify, analyse and group.
+Manually going through data and identifying loans is very time consuming and repetitive. This can also introduce room for human error.
+
+Our Loan Transactions endpoint can identify loans for a company's accounting, banking and commerce integrations.
```
Comment (dcoplowe) [inline docs/assess/loans/loan-transactions:14]:
```suggestion
Our Loan Transactions endpoint can identify loans from a company's accounting, banking and commerce integrations. 
```

File: docs/assess/loans/loan-transactions [hunk 2]
```diff
@@ -0,0 +1,61 @@
+---
+title: Loan Transactions"
+description: "Learn how you can identify a companies loans to better understand their financial health"
+sidebar_label: "Loan transactions endpoint"
+createdAt: "2023-06-15T10:32:29.917Z"
+updatedAt: "2023-06-15T16:35:29.376Z"
+---
+
+## Introduction to Loan Transactions endpoint
+Loan data is often not uniform, spread across a variety of integrations and of different types.
```
Comment (dcoplowe) [inline docs/assess/loans/loan-transactions:10]:
```suggestion
Loan data is often not uniform, spread across a variety of software applications and of different types.
```
Might be cleared to talk about a company's software applications than integrations. An integration allows you to access data from an application.

File: docs/assess/loans/loan-transactions [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+These loans are often hard to identify, analyse and group.
+Manually going through data and identifying loans is very time consuming and repetitive. This can also introduce room for human error.
+
+Our Loan Transactions endpoint can identify loans for a company's accounting, banking and commerce integrations. 
+It can work across a variety of integrations for all three integration types.
+
+The endpoint will you get a better idea of the loan obligations a compnay may have by identifying their loan transactions and loan repyaments.
```
Comment (dcoplowe) [inline docs/assess/loans/loan-transactions:17]:
```suggestion
The endpoint will give analysts a better idea of the loan obligations a company may have by identifying their loan transactions and loan repayments.
```

File: docs/assess/loans/loan-transactions [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+The amount of a loan a company has paid off and the number of loans a company has.
+
+The endpoint can show:
+- The loan drawdowns a company has
+- The loan repayments made by a company
+- The interest rate for a loan drawdown (where possible)
+
+Codat suggests that you make sure a company has been recently synced before hitting the endpoint.
```
Comment (dcoplowe) [inline docs/assess/loans/loan-transactions:25]:
```suggestion
Codat suggests that you make sure a company has been recently synced before calling the endpoint.
```

File: docs/assess/loans/loan-transactions [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+Accounting data connection:
+- JournalEntries
+- Chart of Accounts
+
+Banking data connection:
+- Banking Accounts
+- Banking Transactions
+
+Commerce data connection:
```
Comment (dcoplowe) [inline docs/assess/loans/loan-transactions:38]:
```suggestion
Commerce connection:
```

File: docs/assess/loans/loan-transactions [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+- Commerce Transactions
+
+If the company does not have a any of these connections, then an error will be thrown.
```
Comment (dcoplowe) [inline docs/assess/loans/loan-transactions:41]:
```suggestion
If the company does not have a any of these connections an error will be thrown.
```

Comment (pzaichkina) [inline docs/assess/loans/loan-transactions:41]:
```suggestion
If the company does not have any of these connections, the endpoint will respond with an error.
```

File: docs/assess/loans/loan-transactions [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+Whilst Codat support most integrations, there are a minority which are not supported.
+
+## Invoices Report
+The Loan Transactions endpoint returns a list of a companies loan transactions across all of its valid data connections.
```
Comment (pzaichkina) [inline docs/assess/loans/loan-transactions:46]:
```suggestion
The Loan transactions endpoint returns a list of a company's loan transactions across all of its valid data connections.
```

File: docs/assess/loans/loan-transactions [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+This may include drawdowns, repayments etc.
```
Comment (pzaichkina) [inline docs/assess/loans/loan-transactions:47]: is there anything more than drawdowns and repayments? if the list is short (up to 4 items), we should include the whole list.

File: docs/assess/loans/loan-transactions [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+
+### Accounting
```
Comment (pzaichkina) [inline docs/assess/loans/loan-transactions:49]: Not sure about this split into sections - what is the relation between acctg/commerce/banking and the Invoices report?

File: docs/assess/loans/loan-transactions [hunk 10]
```diff
@@ -0,0 +1,61 @@
+---
+title: Loan Transactions"
+description: "Learn how you can identify a companies loans to better understand their financial health"
+sidebar_label: "Loan transactions endpoint"
+createdAt: "2023-06-15T10:32:29.917Z"
+updatedAt: "2023-06-15T16:35:29.376Z"
+---
+
```
[the remark this answers is not in the document]
Comment (s-ahmed-codat) [reply docs/assess/loans/loan-transactions:8]: Sorry do you mean the entire document?

[the remark this answers is not in the document]
Comment (pzaichkina) [reply docs/assess/loans/loan-transactions:8]: yes, I think the entire document should live in the API reference as its goal is to describe the endpoint. @dcoplowe what do you think?

File: docs/assess/loans/loan-transactions (hunk 9 above)
Comment (s-ahmed-codat) [reply docs/assess/loans/loan-transactions:49]: Depending on what query parameter you use (param = connectionType), the output is slightly different

File: docs/assess/loans/loan-transactions [hunk 11]
```diff
@@ -0,0 +1,45 @@
+---
+title: Loan transactions
+description: "Learn how to identify a company's loans to better understand its financial health"
+sidebar_label: "Loan transactions endpoint"
+createdAt: "2023-06-15T10:32:29.917Z"
+updatedAt: "2023-06-21T16:35:29.376Z"
+---
+
+## Introduction to Loan transactions endpoint
+Loan data is often not uniform, spread across a variety of software applications and the data can be difficult to interpret..
```
Comment (mcclowes) [inline docs/assess/loans/loan-transactions:10]:
```suggestion
Loan data is often not uniform, spread across a variety of software applications and the data can be difficult to interpret.
```

**Comment by pzaichkina** [comment]
@s-ahmed-codat I believe you missed adding this page to the navigation - please check if you want it to be navigable to

**Review by pzaichkina — APPROVED** [review]

**Pull request merged**